GitVersion.yml

```diff
@@ -0,0 +1,12 @@
+next-version: 3.0.0
+mode: ContinuousDeployment
+assembly-informational-format: "{Major}.{Minor}.{Patch}-{PreReleaseTag}"
+branches:
+  main:
+    regex: ^main$
```

Comment:
hmm, also don't know they are related to my previous comment

Reply:
Leaving this ones to have consistent config between our repos (it's the same as speckle-sharp-sdk and connectors).

It is required for gitversion to work properly